rfc-177-edition-all-content.md

```diff
@@ -0,0 +1,148 @@
+---
```

Comment:
General thoughts at the top, so that they can be threaded:

- Sorry it's taken me such a long time to get a chance to look at this 🙇🏻 
- I agree with the thrust of the other reviewers, which I'd summarise as:
    - Having an approved RFC doesn't necessarily mean we're going to prioritise the work to implement it, and the decision on if / when to prioritise the work is best left with the Whitehall team
    - There's some tech debt in Whitehall which it would be nice to clear up before continuing to build things on the foundation provided by the Edition model and its Single Table Inheritance approach
    - We have various ideas about how to clear up that tech debt, but we haven't got a decision on a way forward
    - Other than those concerns, everyone generally agrees that making everything Editionable is a good idea

So generally this is a good idea, but we just have practicalities to work through. That's left us a little unsure of whether to approve and merge the RFC, or whether to postpone it until we're ready to embark on the work.

I guess we could have it as

```
status: approved
implementation: blocked
```

With some note in the RFC explaining why it's blocked?

I wonder if all the non-editionable things are equally blocked - I could imagine that something like TakePartPages might be quite similar to Edition (e.g. it has a body), and therefore not too smelly to extend Edition and follow that pattern. Whereas some of the things that are knitted together like People / Roles / RoleAppointments / Organisations might be a worse fit for the current approach to Editions.

Reply:
That seems like a good idea to me.  I've updated the PR to reflect the blocked implementation status and added some reasons why.  [Link](https://github.com/alphagov/govuk-rfcs/compare/d1882e6c6b0e3b34ac4b1f1e2aed5308dfa13c2b..22aa2871e099b03f5ea56b8e6333b4bb0be13abe) to the diff.